src/sympc/utils/__init__.py

```diff
@@ -20,4 +21,5 @@
     "get_type_from_ring",
     "decompose",
     "RING_SIZE_TO_TYPE",
+    "get_nr_bits",
```

Comment:
OUGHTTO: Could we also test this function?

Reply:
Added a test for the same.